src/main/fc/rc.c

```diff
@@ -344,37 +344,36 @@ bool getRxRateValid(void)
 static FAST_CODE_NOINLINE void rcSmoothingSetFilterCutoffs(rcSmoothingFilter_t *smoothingData)
 {
     // in auto mode, calculate the RC smoothing cutoff from the smoothed Rx link frequency
-    const float minCutoffHz = 15.0f; // don't let any RC smoothing filter cutoff go below 15Hz
-
     const bool autoSetpointSmoothing = smoothingData->setpointCutoffSetting == 0;
     const bool autoThrottleSmoothing = smoothingData->throttleCutoffSetting == 0;
 
-    float setpointCutoffFrequency = smoothingData->setpointCutoffFrequency;
-    float throttleCutoffFrequency = smoothingData->throttleCutoffFrequency;
-    if (autoSetpointSmoothing) {
-        setpointCutoffFrequency = MAX(minCutoffHz, smoothedRxRateHz * smoothingData->autoSmoothnessFactorSetpoint);
-        smoothingData->setpointCutoffFrequency = setpointCutoffFrequency; // update for logging
-    }
+    const float minCutoffHz = 15.0f; // don't let any RC smoothing filter cutoff go below 15Hz
+    const float dT = targetPidLooptime * 1e-6f;
 
-    if (autoThrottleSmoothing) {
-        throttleCutoffFrequency = MAX(minCutoffHz, smoothedRxRateHz * smoothingData->autoSmoothnessFactorThrottle);
-        smoothingData->throttleCutoffFrequency = throttleCutoffFrequency; // update for logging
-    }
+    if (autoSetpointSmoothing) {
```

Comment:
Do we really even need this? Before it was setting the cutoff to just the same value if the RC smoothing was set to a static value. Thats not really a usecase that happens often (pretty much everyone is using the auto setpoint), and its kind of nice to keep the CPU load similar between configurations when reasonable to do.

Reply:
Great suggestion. Applied